string source
string destination
string message
float64 estimated_distance
geometry_msgs/Point estimated_orientation
